packages/docs/docs/guides/6-hosting-a-node/1-docker.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Running a node with Docker
+keywords: [Request node, setup, docker]
+---
+
+Running a Request Node with Docker is easy. There are only a few requirements:
+
+- Docker installed on your system;
+- A web3 provider (we recommend using a service like [infura](infura.io));
+- An ethereum wallet with some funds for gas (if you plan on creating requests through this node);
+
+## Launching the IPFS node
+
+To launch the IPFS node run:
+
+```bash
+docker run -p 5001:5001 -p 4001:4001 requestnetwork/request-ipfs
+```
+
+This command will launch the IPFS node with Request network configurations.
+
+## Launching the Request Node
+
+To launch the Request node you can run:
+
+```bash
+docker run -p 3000:3000 -e MNEMONIC="<your wallet mnemonic>" -e WEB3_PROVIDER_URL="<your web3 provider url>" -e ETHEREUM_NETWORK_ID="<ethereum network id>" -e IPFS_HOST="host.docker.internal"  requestnetwork/request-node
```

Comment:
why not reference the internal docker name?

Reply:
It's not using compose, and setting up a network between them would be more work.